### DISCUSSION TITLE
悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用

### DISCUSSION THREAD

**Discussion by unique48y** [immersive-translate/immersive-translate#1132] [Help]
### 概述 | Summary

悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用

### 补充信息 | Additional information

悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用

**Comment by theowenyoung** [comment]
请问用的是浏览器插件还是油猴脚本？哪个版本，哪个浏览器？截图一下设置页面？

**Comment by unique48y** [comment]
油猴插件&nbsp; 最新版本&nbsp; 谷歌浏览器&nbsp; &nbsp;
关闭了 还是在







------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月20日(星期一) 下午2:28
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [immersive-translate/immersive-translate] 悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用 (Discussion #1132)





 
请问用的是浏览器插件还是油猴脚本？哪个版本，哪个浏览器？截图一下设置页面？

**Reply by theowenyoung** [reply]
<img width="419" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/62473795/937dbd5d-5b60-483a-81ac-4e56e3b7394e">

是点击这里关闭的吗？我测试一切正常。 你会不会是同时安装了油猴和扩展？

**Reply by theowenyoung** [reply]
截图下这个页面：

<img width="1241" alt="image" src="https://github.com/immersive-translate/immersive-translate/assets/62473795/78340c9e-1ef4-4cb5-98b0-832464c3021f">

**Comment by unique48y** [comment]
------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年11月20日(星期一) 晚上8:17
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [immersive-translate/immersive-translate] 悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用 (Discussion #1132)





 
截图下这个页面：

**Comment by unique48y** [comment]
还有一个问题 我绑定了GPT的key 为什么 显示429&nbsp; 我的最大请求已经设置成3了 还是不行







------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "immersive-translate/immersive-translate"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年11月20日(星期一) 晚上8:17
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [immersive-translate/immersive-translate] 悬浮球隐藏不了  在设置里面关闭了 网页还在 刷新也没用 (Discussion #1132)





 
截图下这个页面：

**Reply by theowenyoung** [reply]
建议在openai 账户里绑卡，提升请求频率，不然的话，几乎无法使用。
